src/buttons/Button.js

```diff
@@ -30,11 +30,18 @@ class Button extends Component {
     }
   }
 
+  handleOnPress = () => {
+    const { loading, onPress } = this.props;
+
+    if (!loading) {
+      onPress();
+    }
+  };
+
   render() {
     const {
       TouchableComponent,
       containerStyle,
-      onPress,
```

Comment:
This won't work. Removing onPress from here means it'll be in `...attributes`. Which means your onPress handler will be overridden and `handleOnPress` will never run

Reply:
I have corrected it